src/search.cpp

```diff
@@ -1319,14 +1319,14 @@ namespace {
 #endif
 #ifdef HORDE
         if (pos.is_horde() && pos.is_horde_loss())
-            bestValue = excludedMove ? alpha
-            : pos.is_suicide() ? pos.suicide_stalemate(ss->ply, DrawValue[pos.side_to_move()])
-            : mate_in(ss->ply+1);
+            bestValue = excludedMove ? alpha : mate_in(ss->ply+1);
```

Comment:
Affirmative, must be `mated_in(ss->ply)`, This is _my_ glitch. Sorry and thank you. Will update accordingly.